src/skmultiflow/lazy/sam_knn.py

```diff
@@ -205,22 +207,30 @@ def size_check_fade_out(self):
         return STMShortened
 
     def size_check_STMLTM(self):
-        """Makes sure that the STM and LTM combined doe not surpass the maximum size, only used when use_ltm=True."""
+        """Makes sure that the STM and LTM combined doe not surpass the maximum size,
+        only used when use_ltm=True."""
         STMShortened = False
         if len(self._STMLabels) + len(self._LTMLabels) > self.maxSTMSize + self.maxLTMSize:
             if len(self._LTMLabels) > self.maxLTMSize:
-                self._LTMSamples, self._LTMLabels = self.cluster_down(self._LTMSamples, self._LTMLabels)
+                self._LTMSamples, self._LTMLabels = self.cluster_down(
+                    self._LTMSamples, self._LTMLabels)
             else:
                 if len(self._STMLabels) + len(self._LTMLabels) > self.maxSTMSize + self.maxLTMSize:
                     STMShortened = True
                     numShifts = int(self.maxLTMSize - len(self._LTMLabels) + 1)
                     shiftRange = range(numShifts)
-                    self._LTMSamples = np.vstack([self._LTMSamples, self._STMSamples[:numShifts, :]])
-                    self._LTMLabels = np.append(self._LTMLabels, self._STMLabels[:numShifts])
-                    self._LTMSamples, self._LTMLabels = self.cluster_down(self._LTMSamples, self._LTMLabels)
+                    self._LTMSamples = np.vstack(
+                        [self._LTMSamples, self._STMSamples[:numShifts, :]])
+                    self._LTMLabels = np.append(self._LTMLabels,
+                                                self._STMLabels[:numShifts])
+                    self._LTMSamples, self._LTMLabels = self.cluster_down(
+                        self._LTMSamples, self._LTMLabels)
                     self._STMSamples = np.delete(self._STMSamples, shiftRange, 0)
                     self._STMLabels = np.delete(self._STMLabels, shiftRange, 0)
-                    self.STMDistances[:len(self._STMLabels),:len(self._STMLabels)] = self.STMDistances[numShifts:len(self._STMLabels)+numShifts, numShifts:len(self._STMLabels)+numShifts]
+                    self.STMDistances[:len(self._STMLabels),
+                    :len(self._STMLabels)] = self.STMDistances[  # noqa: E128
+                                             numShifts:len(self._STMLabels) + numShifts,
+                                             numShifts:len(self._STMLabels) + numShifts]
```

Comment:
Tough case here. Would using the following strategy exceed the length limit?

`self.STMDistances[:len(self._STMLabels), :len(self._STMLabels)] = \`

Reply:
This way is better: 
```python
                    self.STMDistances[:len(self._STMLabels), :len(self._STMLabels)] = \
                        self.STMDistances[numShifts:len(self._STMLabels) + numShifts,
                                          numShifts:len(self._STMLabels) + numShifts]
```